README.md

```diff
@@ -48,77 +40,29 @@ ebcli help
-# Create genesis transaction
-ebd gentx --name validator
-# Enter password
+- **Initialization**: setup the Bridge chain, add accounts, start the Bridge chain, and test available commands
+- **Setup Peggy locally**: start local Ethereum blockchain, compile and deploy contracts, activate the contracts, and get the registry contract's deployed address 
+- **Ethereum to Cosmos asset transfers**: start the Relayer service, send lock transaction containing local assets to the contracts, and test ERC20 support
+- **Using Peggy with the Ropsten testnet**: setup interfacting with the Ropsten testnet, deploy contracts to Ropsten testnet, start the Relayer service on the Ropsten testnet, and send lock transaction containing Ropsten testnet assets to the contracts
+- **Cosmos to Ethereum asset transfers**: setup interfacing with tendermint, start the Relayer service, start the Oracle Claim Relayer service, burn assets on tendermint, create prophecy and oracle claims on Ethereum, and process prophecy claims 
 
-# Collect genesis transaction
-ebd collect-gentxs
+## Initialization
 
-# Now its safe to start `ebd`
-ebd start
+In order to facilitate cross chain transfers, the Bridge blockchain must be set up by following these [steps](./docs/initialization.md).
 
-# Then, wait 10 seconds and in another terminal window, test things are ok by sending 10 tok tokens from the validator to the testuser
-ebcli tx send validator $(ebcli keys show testuser -a) 10stake --chain-id=peggy --yes
+## Setup Peggy locally
+To testing the transfer of Ethereum based assets, set up and start a local Ethereum chain by following these [steps](./docs/local-ethereum-usage.md).
 
-# Wait a few seconds for confirmation, then confirm token balances have changed appropriately
-ebcli query account $(ebcli keys show validator -a) --trust-node
-ebcli query account $(ebcli keys show testuser -a) --trust-node
+## Ethereum to Cosmos asset transfers
+With a local Ethereum blockchain running, you can participate in Ethereum -> Cosmos asset transfers by starting the Relayer service and acting as a validator. Validators witness the locking of Ethereum/ERC20 assets and sign a data package containing information about the lock, which is then relayed to tendermint and witnessed by the EthBridge module. Once other validators have confirmed that the transaction's information is valid, the funds released by the Oracle module and transferred to the intended recipient's address. In this way, Ethereum assets can be transferred to Cosmos-SDK based blockchains. The process is described [here](./docs/ethereum-to-cosmos.md).
 
-# Then wait 10 seconds then confirm your validator was created correctly, and has become Bonded status
-ebcli query staking validators --trust-node
+## Using Peggy with the Ropsten testnet
+Instead of transferring local Ethereum assets to Cosmos-SDK based blockchains, you can test out transferring rEth from the Ropsten testnet by following these [steps](./docs/ropsten-testnet-usage.md).
```

Comment:
```suggestion

Instead of transferring local Ethereum assets to Cosmos-SDK based blockchains, you can test out transferring `rEth` from the Ropsten testnet by following these [steps](./docs/ropsten-testnet-usage.md).
```

Reply:
https://github.com/cosmos/peggy/pull/104/commits/cff943a3b789116e3459f065a41a01e93b4fa73b